tests/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import validate_email
 
-# write your tests here
+
```

Comment:
The given function contains bugs. Write tests that would be failed

Reply:
Fixed!